presentation/src/main/java/me/androidbox/presentation/event/viewmodel/EventViewModel.kt

```diff
@@ -63,6 +63,22 @@ class EventViewModel @Inject constructor(
                     )
                 }
             }
+            is EventScreenEvent.OnSaveEventDetails -> {
+                _eventScreenState.update { eventScreenState ->
+                    eventScreenState.copy(
+                        saveEventDetails = eventScreenEvent.event
```

Comment:
Wouldn't you have this Event on the state? Why are you passing it through an Event (EventScreenEvent.OnSaveEventDetails) ?

Reply:
@MKiperszmid not sure I understand. This Event will contain the event information that has been set by the user. Then will be saved to the room dB. What should I be doing here?

Reply:
<img width="599" alt="image" src="https://user-images.githubusercontent.com/16141845/228954634-9d2864c4-e0d7-4f72-8a06-7fceefacc232.png">
Yeah but you have all the info to create the Event in the State

So why pass the Event over the Screen to the ViewModel, if the viewmodel already has all the info?